app/models/booking_type.rb

```diff
@@ -1,9 +1,12 @@
 
 class BookingType < ActiveRecord::Base
-	has_many :order_transports
+  has_many :order_transports
 
-	def self.appointment
-		BookingType.find_by(identifier: "appointment")
-	end
+  def self.appointment
+    BookingType.find_by(identifier: "appointment")
```

Comment:
Can be written as `find_by(identifier: "appointment")` instead of ` BookingType.find_by(identifier: "appointment")`. It is class method thus self will be class here and we can directly use find_by. :)

CC: @steveyken